src/github.com/getlantern/rotator/daily_rotator_test.go

```diff
@@ -1,10 +1,11 @@
 package rotator
 
 import (
-	"github.com/stretchr/testify/assert"
 	"os"
 	"testing"
 	"time"
+
+	"github.com/getlantern/testify/assert"
```

Comment:
We already use it in a bunch of other places, so this way we don't have two versions of the thing.